upstream.wrap

```diff
@@ -0,0 +1,7 @@
+[wrap-file]
+directory = nlohmann_json-3.4.0
```

Comment:
This is incorrect, please try `include` or something else. There is a tool that you can try to use to test your PR locally:
`mesonwrap review nlohmann_json --pull_request=3`. It is available from https://github.com/mesonbuild/wrapweb

Reply:
I updated my branch to fix this.

``mesonwrap review nlohmann_json --pull_request=3`` now seems fine with the ``directory`` but it stills returns 1.

Local log:
```
$ mesonwrap review nlohmann_json --pull_request=3
Inspecting project nlohmann_json, branch 3.4.0.
Repo name valid: ☑
Branch name valid: ☑
Target branch is not master: ☑
Has readme.txt: ☑
Has LICENSE.build: ☑
Has upstream.wrap: ☑
Repo contains only buildsystem files: ☑
upstream.wrap has directory: ☑
upstream.wrap directory has no path separators: ☑
upstream.wrap has source_url: ☑
upstream.wrap has source_filename: ☑
upstream.wrap source_filename has no path separators: ☑
upstream.wrap has source_hash: ☑
Download URL works: ☑
Hash matches: ☑
upstream.wrap directory 'include' exists: ☑
Patch merges with source: ☑
The Meson build system
Version: 0.48.2
Source dir: /tmp/tmphwf8i8nd/src/include
Build dir: /tmp/tmphwf8i8nd/bin
Build type: native build
Project name: nlohmann_json
Project version: 3.4.0
Native C++ compiler: /usr/bin/clang++ (clang 7.0.0 "clang version 7.0.0 (tags/RELEASE_700/final)")
Build machine cpu family: x86_64
Build machine cpu: x86_64
Build targets in project: 0
WARNING: Trying to use none sanitizer on Clang with b_lundef.
This will probably not work.
Try setting b_lundef to false instead.
Found ninja-1.8.2 at /usr/bin/ninja
meson setup: ☑
ninja: Entering directory `/tmp/tmphwf8i8nd/bin'
[0/1] Running all tests.
No tests defined.
ninja test: ☑
<mesonwrap.cli.Command object at 0x7fec144f2ac8>

$ echo $?
1
```

Reply:
I applied undocumented `lead_directory_missing` which is the proper way to do this.